Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/Mappings.kt

```diff
@@ -31,7 +31,8 @@ fun DeclarationMapper.isMappedToSigned(integerType: IntegerType): Boolean = inte
 
 fun DeclarationMapper.getKotlinClassFor(
         objCClassOrProtocol: ObjCClassOrProtocol,
-        isMeta: Boolean = false
+        isMeta: Boolean = false,
+        isJ2ObjC: Boolean = false
```

Comment:
Had to create this variable and pass it down 2 methods in order to fix naming so I would get "myInterface" instead of "myInterfaceProtocol". Is this OK?

Reply:
Isn't this the same as the name override used in the target class?

Reply:
nameOverride was for methods, and it was simple as I could just add it to the implementation I create. This is instead part of Container(class/protocol) naming, and doesn't follow the same path.

Reply:
Call this useNameForProtocol or some such